test/vm-customize

```diff
@@ -37,6 +37,7 @@ parser.add_argument('-r', '--run-command', action='append', dest="commandlist",
                                            default=[], help='Run command inside virtual machine')
 parser.add_argument('-s', '--script', action='append', dest="scriptlist",
                                       default=[], help='Run selected script inside virtual machine')
+parser.add_argument('-u', '--upload', action='append', dest="uploadlist", default=[], help='Upload files to destination dir separated by : EXAMPLE: -u file.txt:/var/lib')
```

Comment:
Also, it doesn't have to be a destination dir. For single files, it could also be a filename. The underlying command is `scp`, so that's the behavior to expect. Also, it copies recursively, so it can upload a file or a directory.